kiosk/kiosk_browser/browser_widget.py

```diff
@@ -44,6 +54,15 @@ def _load_finished(self, success):
         if not success:
             QTimer.singleShot(5000, self.reload)
 
+    def _proxy_auth(self, get_current_proxy, url, auth, proxyHost):
+        proxy = get_current_proxy()
+        if proxy is not None and proxy.username is not None and proxy.password is not None:
+            logging.info("Authenticating proxy")
+            auth.setUser(proxy.username)
+            auth.setPassword(proxy.password)
+        else:
+            logging.info("Ignoring proxy authentication")
```

Comment:
I am not sure I understand what this log says. Reading the conditional above I get the impression that no credentials were given, but this log reads like the kiosk somehow chose to ignore the credentials.

Reply:
Adapting the log in https://github.com/dividat/playos/pull/54/commits/811d5c06647486393f4cdf76aae3a041fecfb4ff